client/post-editor/drafts-button/index.jsx

```diff
@@ -10,12 +10,12 @@ import { translate } from 'i18n-calypso';
  */
 import Button from 'components/button';
 import Count from 'components/count';
-import QueryPostCounts from 'components/data/query-post-counts';
+import QueryPosts from 'components/data/query-posts';
 import { getSelectedSiteId } from 'state/ui/selectors';
 import { isJetpackSite } from 'state/sites/selectors';
-import { getAllPostCount } from 'state/posts/counts/selectors';
```

Comment:
I think we should only show your drafts here, anyways. We shouldn't count how many posts we have in `state.posts` because it may not represent the total — that's why we have a separate counts tree.